src/utils/validation/index.ts

```diff
@@ -27,41 +28,56 @@ export function isValidAssetID(assetText: string) {
 }
 
 // validation for send quantity inputs
-export function isValidQuantity(quantity: string) {
-  // ensure only a single decimal
-  if (quantity.split(".").length > 2) {
-    console.error("quantity entered has multiple decimals");
-    return false;
-  }
+export function isValidQuantity(quantity: string | BigNumber) {
+  quantity = bnToString(quantity);
 
   // ensure value is a string
   if (typeof quantity !== "string") {
     console.error("quantity entered is not a string");
     return false;
   }
 
+  // ensure only a single decimal
+  if (quantity.split(".").length > 2) {
```

Comment:
i'm going to run this in the next few days and see if i can break it with some bad inputs. that'll give me confidence that it's working. would be great if i did it with cypress so you had a working test to go with it.

Reply:
great idea!

Reply:
upon further review of the bignumber code, tostring doesn't seem to offer up any way to make two decimals exist by using a really large number or really small. see https://mikemcl.github.io/bignumber.js/#toS for details